app/summit-2016/data/schedule.json

```diff
@@ -1,162 +1,142 @@
 [
   {
-    "date":"September 14th",
+    "date":"October 17th",
     "day":"Monday",
     "sessions":[
       {
-        "name":"Code Labs and Pre-registration",
-        "speaker":"",
-        "time":"Monday 5pm - 9pm",
-        "desc":"Polymer code labs provide a guided, tutorial, hands-on coding experience. Learn and build elements and applications with the Polymer Team. **BYOL (Bring Your Own Laptop)**"
-      }
-    ]
-  },
-  {
-    "date":"September 15th",
-    "day":"Tuesday",
-    "sessions":[
+        "name":"Opening Keynote",
+        "speaker":"Taylor Savage",
+        "desc":"Welcome to the second Polymer Summit! We'll kick off with a whirlwind tour of what's new in the world of Web Components and the Polymer Project."
+      },
       {
-        "name":"Registration Opens",
-        "time":"Tuesday @ 7:30am"
+        "name": "The Way of the Web - ING",
+        "speaker": "Ben Issa",
+        "desc": "An end-to-end view on how ING builds beautiful experiences for its customers using Web Components."
```

Comment:
Need to capitalize or not capitalize "web components" consistently in all the descriptions. We have a couple with caps at the top, some with no caps later on.

In the docs, we've tried to go with no caps (unless we're talking about a specification by name). But I don't think consistency with the rest of the docs is as important as consistency across the schedule.